MatchedTransitions/MatchedTransitionsDemo/Screens/Details/DetailsScreen.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  DetailsScreen.swift
+//  MatchedTransitions
+//
+//  Created by Igor Kharytaniuk on 20.07.24.
+//
+
+import SwiftUI
+
+struct DetailsScreen: View {
+    let book: Book
+    var body: some View {
+        VStack(alignment: .center,spacing: 32) {
+            if let imagePath = book.data.image {
+                if let cachedImage = DataManager.shared.imageCache[imagePath] {
+                    cachedImage
+                        .resizable()
+                        .scaledToFit()
+                        .frame(width: 250, height: 250)
+                        .matchedGeometry(id: "image_\(book.id)", isSource: false)
```

Comment:
In order for the transition to work properly the modifier should be better applied to the view instead of the modified frame.
```suggestion
                        .matchedGeometry(id: "image_\(book.id)", isSource: false)
                        .frame(width: 250, height: 250)
```

Reply:
I see in the readme it is wrongly set after the `frame` modifier, maybe you could update it there as well 🙂